library/src/main/java/org/xmtp/android/library/Contacts.kt

```diff
@@ -29,6 +30,13 @@ data class Contacts(
         for (envelope in response.envelopesList) {
             knownBundles[peerAddress] = ContactBundleBuilder.buildFromEnvelope(envelope)
```

Comment:
We can filter in the for loop, so the code can tolerate a few bad bundles and still return the first legit one.

Reply:
Ahh good call. I fixed that in https://github.com/xmtp/xmtp-android/pull/53/commits/b6ead02697418abcddf625a4f1da55b2e8e8573c